tango.pc.cmake

```diff
@@ -2,10 +2,11 @@ prefix = @CMAKE_INSTALL_PREFIX@
 exec_prefix = ${prefix}
 includedir = ${prefix}/include
 libdir = ${prefix}/@CMAKE_INSTALL_LIBDIR@
+devices_dir = ${prefix}/@CMAKE_INSTALL_LIBDIR@/tango
```

Comment:
For your information the path where is installed the device server is different between deb and rpm. 
For the hundreds of rpm we have the default path is: /usr/bin

Will it be possible to easily change the default directory following the platform?